backend/app/Models/FinishedLesson.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Models;
+ 
+use Illuminate\Database\Eloquent\Relations\Pivot;
+use Illuminate\Database\Eloquent\Model;
+
+class FinishedLesson extends Pivot
+{
+    protected $table = 'finished_lessons';
```

Comment:
@rogeliojohnoliverio no need for this lines since id is auto incremented already by default, same with id. 
```
protected $table = 'finished_lessons';
protected $primaryKey = 'id';
public $incrementing = true;
```

Reply:
@roseaugusto hi miss.. umm as i have researched miss since this is a customize model for pivot table, need daw sya butngan og incrementing=true since by default false sya and dli makita ang id ani na pivot table if dli iset to true